Sources/Datadog/CrashReporting/CrashReporter.swift

```diff
@@ -135,9 +137,8 @@ internal class CrashReporter {
                 """
                 Failed to encode crash report context. The app state information associated with eventual crash
                 report may be not in sync with the current state of the application.
-
-                Error details: \(error)
-                """
+                """,
+                error: error
```

Comment:
`userLogger` [is configured with `.short` format](https://github.com/DataDog/dd-sdk-ios/blob/79c63be50b4e5f1ec3e6432e84da34cac30efd07/Sources/Datadog/Utils/InternalLoggers.swift#L58) meaning that it [only prints the `message` to the console](https://github.com/DataDog/dd-sdk-ios/blob/79c63be50b4e5f1ec3e6432e84da34cac30efd07/Sources/Datadog/Logging/LogOutputs/LogConsoleOutput.swift#L79), hence emitting `error` to the `message` has more sense than setting `error: error` which won't be displayed to the user.

Reply:
BTW, we could add an improvement (to the backlog) to make `.short` format also include `error.message` (if available). That would make more sense IMO 👌.

Reply:
Ah that's why, I thought this was a mistake 😆 I will revert then.
> Makes sense to include the error in short format, I will log something!